hedera-node/src/test/java/com/hedera/services/contracts/execution/CallEvmTxProcessorTest.java

```diff
@@ -143,8 +144,11 @@ void assertSuccessExecution() {
 		given(globalDynamicProperties.fundingAccount()).willReturn(new Id(0, 0, 1010).asGrpcAccount());
 		given(aliasManager.resolveForEvm(receiverAddress)).willReturn(receiverAddress);
 		given(storageExpiry.hapiCallOracle()).willReturn(oracle);
+		given(globalDynamicProperties.chainIdBytes32()).willReturn(Bytes32.ZERO);
```

Comment:
All the test  return `Bytes32.ZERO` on chainId get.
is there coverage for the valid values for the 4 env types (local, previewnet, testnet and mainnet) possibilities?

Reply:
Same issue in `CallLocalEvmTxProcessor` and `CreateEvmTxProcessor` test classes

Reply:
Behavior of this class won't change based on the `chainId`; we just need to test that its getting the value returned by `dynamicProperties`.

Reply:
Okay